pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
   "psutil",
   "strawberry-graphql==0.208.2",
   "pyarrow",
-  "typing-extensions",
+  "typing-extensions>=4.5, <5",
```

Comment:
note that strawberry has typing-extension>=4.5 since [0.187.1](https://github.com/strawberry-graphql/strawberry/releases/tag/0.187.1) (6/21/23)